packages/types/lib/sync/api.ts

```diff
@@ -1,4 +1,21 @@
-import type { ApiError, Endpoint } from '../api.js';
+import type { ApiError, Endpoint, ResDefaultErrors } from '../api.js';
+
+export type PostPublicTrigger = Endpoint<{
+    Method: 'POST';
+    Path: '/sync/trigger';
+    Body: {
+        syncs: (string | { name: string; variant: string })[];
+        full_resync?: boolean | undefined;
+        provider_config_key?: string | undefined;
+        connection_id?: string | undefined;
+    };
+    Headers: {
+        'provider-config-key'?: string | undefined;
+        'connection-id'?: string | undefined;
+    };
+    Success: { success: boolean };
+    Error: ResDefaultErrors;
```

Comment:
It's already the default value, so you don't even need to put that ☺️ 
Note that if you want custom code you can do `Error: ApiError<'missing_connection_id'>` and it will extend the default not override 👍🏻